apps/api/src/prisma/prisma.health.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@nestjs/common';
+import { HealthIndicator, HealthIndicatorResult, HealthCheckError } from '@nestjs/terminus';
+import { PrismaService } from './prisma.service'
+
+@Injectable()
+export class PrismaHealthIndicator extends HealthIndicator {
+  constructor(
+    private prisma: PrismaService,
+  ) {
+    super()
+  }
+
+  async isHealthy(key: string): Promise<HealthIndicatorResult> {
+    try {
+      await this.prisma.$queryRaw`SELECT 1`
```

Comment:
this was suggested [here](https://docs.nestjs.com/recipes/terminus#typeorm-health-indicator)